assets/js/util/rest.js

```diff
@@ -51,9 +50,15 @@ const headers = async () => {
   if (organizationId) {
     Object.assign(headerParams, { organization: organizationId })
   }
-  const tokenClaims = await getIdTokenClaims();
```

Comment:
this generate a new token everytime, and thus never expires. This is why in the future state, I only use it for the first rest api call and use the stored token in global state once apolloClient has been created so we can keep using the same token.